html/user/lookup_account.php

```diff
@@ -23,6 +23,14 @@
 require_once("../inc/email.inc");
 require_once("../inc/xml.inc");
 require_once("../inc/ldap.inc");
+require_once("../inc/password.php");
+
+function do_passwd_rehash($user,$passwd_hash) {
+    $database_passwd_hash = password_hash($passwd_hash , PASSWORD_DEFAULT);
+    $result = $user->update(
+        "passwd_hash='$database_passwd_hash'"
```

Comment:
[Here](https://github.com/BOINC/boinc/pull/2401/files#diff-2ebf2ee712812c0682531ca5ebf7c713R73) you escape the password hash before its use as part of an SQL clause. If you think it's required, it should be done here as well.

Reply:
After reviewing, the hash does not need to be escaped so I have removed it from the referenced line.